src/lib/taxcalc.js

```diff
@@ -20,7 +20,7 @@ function isSpecialPriceActive (fromDate, toDate) {
   }
 }
 
-export function updateProductPrices (product, rate, sourcePriceInclTax = false, deprecatedPriceFieldsSupport = false, finalPriceInclTax = true) {
+export function updateProductPrices ({ product, rate, sourcePriceInclTax = false, deprecatedPriceFieldsSupport = false, finalPriceInclTax = true }) {
```

Comment:
@andrzejewsky please add a note about that breaking change in the changelog. There might be some third-party platforms that also are using this method.